### ISSUE TITLE
Feature: End-to-end TensorFlow Extended (TFX) example for TF 2.0.

### ISSUE THREAD

**Issue by dynamicwebpaige** [tensorflow/tensorflow#25369]
**[TensorFlow Extended (TFX)](https://www.tensorflow.org/tfx/)** is an end-to-end, TensorFlow-based, general-purpose machine learning platform implemented at Google. We've already open sourced some TFX libraries with the rest of the system to come. For an overview of TFX, read our [KDD’2017 paper](https://dl.acm.org/citation.cfm?id=3098021) and watch the [Dev Summit talk](https://www.youtube.com/watch?v=vdG7uKQ2eKk).

TFX includes:
- **[TensorFlow Data Validation](https://www.tensorflow.org/tfx/data_validation)**: a library for exploring and validating machine learning data.
- **[TensorFlow Transform](https://www.tensorflow.org/tfx/transform)**: full-pass analyses over data to create transformation graphs that are consistently applied during training and serving.
- **[TensorFlow Model Analysis](https://www.tensorflow.org/tfx/model_analysis)**: libraries and visualization components to compute full-pass and sliced model metrics over large datasets, and analyze them in a notebook.
- **[TensorFlow Serving](https://www.tensorflow.org/serving)**: a flexible, high-performance serving system for machine learning models, designed for production environments.

The purpose of this issue is to track the creation of an end-to-end TFX example using TensorFlow 2.0: a Keras model working with Transform, TFMA, data validation, and Serving.

**Comment by newpoo** [comment]
What's the latest status?

**Comment by gregmalley** [comment]
Nice intro at GoogleNext19 but will a full demo be available soon?

**Comment by ahmedanis03** [comment]
how it is done in the project tracker and still open in github? it is a bit confusing to track the progress.

and please if it is done, how can we access this example?

**Comment by shubham-samant** [comment]
here is an link to the end-to-end example using the NYC taxi dataset - 
https://github.com/tensorflow/tfx/tree/master/tfx/examples/chicago_taxi_pipeline

**Comment by Efaq** [comment]
@shubham-samant this link is sending me to the old example, still in TF 1.X. Do you have a reference for an example using TF 2.0 and TFX?

**Comment by Efaq** [comment]
as @ahmedanis03 mentioned, it is a bit confusing that this is open here but the task is marked as done. Is there already material or is it under development?

**Comment by jvmancuso** [comment]
@dynamicwebpaige, are we to interpret https://github.com/tensorflow/tfx/issues/542 and the current status being "Done" to mean that this will not be a part of the TF 2.0 launch?

**Comment by Efaq** [comment]
I wonder if there are any news regarding that, and specifically regarding TF Transform together with the new Tensorflow 2.0. The functionality in TFT is quite important to produce well-structured preprocessing steps without leaking info between train and validation/test set.

On the other hand, anyone who know about a nice approach to replace TFT for the while?

Thanks

**Comment by maximveksler** [comment]
@dynamicwebpaige @shubham-samant I see some progress and commits being made on the link that was provided. Could you please clarify the status of this and when should we expect this to be "DONE" ?

**Comment by maximveksler** [comment]
> anyone who know about a nice approach to replace TFT for the while

@Efaq i might be able to help but first, could you please clarify what you meant by the following statement and what is the failure condition under which this is expected?

> without leaking info between train and validation/test set


Thanks..

**Comment by rcrowe-google** [comment]
Closing old issue, we have several e2e examples now.
